kolibri/plugins/coach/assets/src/modules/coachNotifications/index.js

```diff
@@ -87,7 +91,7 @@ export default {
     },
     moreNotificationsForClass(store, params) {
       const classroomId = store.state.currentClassroomId;
-      const lastNotification = store.state.notifications.slice(-1)[0];
```

Comment:
If our data is ordered by timestamp, do we need to make this change?

Reply:
I was afraid because I saw that to get the maxId (for the `after` prop) we were using the maxBy, and made me doubt :sweat_smile:. But yes, since we are sorting them in the `INSERT_NOTIFICATIONS` action, they are already sorted. I will change this